presto-docs/src/main/sphinx/release/release-0.271.rst

```diff
@@ -1,37 +1,36 @@
 =============
-Release 0.271
+Release 0.272
```

Comment:
Why are we making changes to the 271 file?

Reply:
Mistake - I had it open since IntelliJ kept closing release-0.272.rst while I was working with Git